{"name": "test_Days[2022-2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 28 == 31", "trace": "self = <TestYear.Test object at 0x0000028E98273400>, year = 2022, month = 2\n\n    @pytest.mark.parametrize(\"year,month\",[(2022,1),(2022,2),(2020,2),(2022,10)])\n    def test_Days(self,year,month):\n        result = year_month(year,month)\n>       assert result == 31\nE       assert 28 == 31\n\nTestYear.py:28: AssertionError"}, "attachments": [{"name": "stdout", "source": "ad3c84bc-9119-4b3d-ad02-8f3f123f1fee-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "year", "value": "2022"}, {"name": "month", "value": "2"}], "start": 1665925791060, "stop": 1665925791060, "uuid": "996173d7-27df-457e-ae23-272823dff5ae", "historyId": "f0dcda44f60aa088b019ce06886d384e", "testCaseId": "bce9ad11c85e3ad0c3ea334c9cd45d98", "fullName": "TestYear.Test#test_Days", "labels": [{"name": "suite", "value": "TestYear"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "DESKTOP-0AC7NBS"}, {"name": "thread", "value": "41744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestYear"}]}